File: solutions/uri/2311/2311.cpp
#include <iostream>
#include <iomanip>

int main() {
    int n, i;
    double difficulty, max_score, min_score, current_score, total_score;
    std::string name;

    std::cin >> n;
    while (n--) {
        max_score = -1;
        min_score = 11;
        total_score = 0.0;

        std::cin >> name;
        std::cout << name << " ";
        std::cin >> difficulty;

        for (i = 0; i < 7; i++) {
            std::cin >> current_score;
            total_score += current_score;
            max_score = std::max(max_score, current_score);
            min_score = std::min(min_score, current_score);
        }

        std::cout << std::fixed << std::setprecision(2);
        std::cout <<
            (total_score - max_score - min_score) * difficulty <<
            std::endl;
    }

    return 0;
}

File: solutions/beecrowd/2311/2311.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

n = int(input())
for _ in range(n):
    max_score = -1
    min_score = 11
    total_score = 0.0

    name = input().strip()
    difficulty = float(input())

    scores = map(float, input().split())

    for score in scores:
        total_score += score
        max_score = max(max_score, score)
        min_score = min(min_score, score)

    output(
        f'{name} {(total_score - max_score - min_score) * difficulty:.2f}\n'
    )
